perun-openapi/openapi.yml

```diff
@@ -17164,6 +17231,22 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/servicesManager/removeDestinations:
```

Comment:
After looking at 'remove-destination-dialog' i realised that we need to remove destinations connected to different services, so we probably need to pass RichDestinations to this method.

Reply:
Good point. As I mentioned on the Daily, in the front-end part you will have to replace also getDestionation() method by getRichDestrionation() method due to these changes - in the `delete-service-from-facility` component.